.github/ISSUE_TEMPLATE/icon_template.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Icon request
+about: Request a new icon
+---
+
+<!--
+  ### IMPORTANT SECURITY NOTE ###
+
+  When opening issues, be sure NOT to include any private or personal
+  information such as secrets, passwords, or any source code that involves
+  data retrieval.
+
+  Also, do not include links to sites on staging.
+-->
+
+<!-- 
+  ### TDS Timeline for releasing icons ###
+
+  New icon requests will be released midway in the next month.
+
+  Example 1: if you submit an icon request and it gets approved on August 3, it will be
+  released on September 15.
+
+  Example 2: if you submit an icon request and it gets approved on August 28, it will be
+  released on September 15.
+-->
+
+## Actions
+
+1. Did you check to see if there are decorative icons that can be used in TDS instead? Yes/No
```

Comment:
Let's take out the word 'decorative' to make this generic and it'll be up to the team to decide/follow up on what type of icon they are asking for